src/05_lists.py

```diff
@@ -8,22 +8,30 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE
+x.extend(y)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
 # YOUR CODE HERE
+x.remove(8)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 99, 10]
 # YOUR CODE HERE
+targetIndex = x.index(10)
+x.insert(targetIndex, 99)
```

Comment:
Great Job! 
You also could have solved it like this: 

x.insert(5, 99)
print(x)